sections/04_study.md

```diff
@@ -220,13 +220,19 @@ prediction at the sequence level, they often cannot predict binding patterns for
 cell types or conditions that have not been previously studied. One solution
 could be to introduce a multimodal model that, in addition to sequence data,
 incorporates cell-line specific features such as chromatin accessibility, DNA
-methylation, or gene expression. Without cell-specific features, another
-solution could be to use domain adaptation methods where the model trains on a
-source cell type and uses unsupervised feature extraction methods to predict on
-a target cell type. TFImpute [@tag:Qin2017_onehot] predicts binding in new cell
-type-TF pairs, but the cell types must be in the training set for other TFs.
-This is a step in the right direction, but a more general domain transfer model
-across cell types would be more useful.
+methylation, or gene expression. Without cell-specific features, we need models
+which can handle the differences between the domains. For example, if we have
+the ChIP-seq data for a particular TF from a mouse, but we want to predict the
+binding locations for that same TF on a human, we need to transfer the knowledge
+of training on the mouse to the human. This has been done in many other areas
+such as sentiment analysis [@doi:10.1.1.231.3442]. For example, we can train on
```

Comment:
This isn't a valid DOI.  Can you please change the citation to `[@url:http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.231.3442]`?